client-core/src/wallet/syncer.rs

```diff
@@ -547,9 +547,31 @@ impl<
     }
 }
 
+/// testnet v0.5
+const GENESIS_APP_HASH: &str = "F62DDB49D7EB8ED0883C735A0FB7DE7F2A3FA322FCD2AA832F452A62B38607D5";
+
+fn check_genesis_hash(genesis_hash: &[u8]) -> Result<()> {
+    let hash_setted = std::env::var("GENESIS_APP_HASH").unwrap_or_else(|_| GENESIS_APP_HASH.into());
```

Comment:
it would be good to list this extra environment variable in the command line help, as it lists all the other client environment variables

Reply:
yes, that would be good feature